sites/includes/canonical.html

```diff
@@ -0,0 +1,10 @@
+{% comment %}
+Should only be called if page.altLangPage is set and a BasePath URL exist
+{% endcomment %}
+{%- if site.baseurl -%}
+<link rel="canonical" href="{{ page.url | prepend: site.baseurl }}" />
+<link rel="alternate" href="{{ page.altLangPage | prepend: site.baseurl | prepend: i18nText-altLang }}" hreflang="{{ i18nText-altLang }}" />
+{% elsif site.urlalt %}
+<link rel="canonical" href="{{ page.url | prepend: site.urlalt[ page.lang ] }}" />
+<link rel="alternate" href="{{ page.altLangPage | prepend: site.urlalt[ i18nText-altLang ] }}" hreflang="{{ i18nText-altLang }}" />
```

Comment:
`canonical` should represent the final URL of a given page, not the base url or the site neither the alternative URL of the site with `alternate` + `hreflang`.

Those 2 meta are page specific, not site specific.

Note: A permanent URL, if exist, can be seen as the canonical URL.

Reply:
That's why I wanted to concatenate the base url with the page file name, but the logic still need works. I wanted to take into account that the base path could be the same url for both language or different for both. But that requires some custom properties in the `_config.yml` file and sounds like overhead to me.

Reply:
Sorry, I missed the "prepend" instruction for site.baseurl.

Don't assume this language toggle: prepend: i18nText-altLang

I don't think the site.urlalt is needed for now. Let build gcweb-jekyll for experimentation for now and let deal with that when we know that some production site use it.

Reply:
As discussed, canonical link would be in a different PR